internal/clientconn/conninfo/conn_info.go

```diff
@@ -116,7 +114,7 @@ func (connInfo *ConnInfo) Cursor(id int64) *Cursor {
 }
 
 // StoreCursor stores cursor and return its ID.
-func (connInfo *ConnInfo) StoreCursor(iter iterator.Interface[int, *types.Document], filter *types.Document) int64 {
+func (connInfo *ConnInfo) StoreCursor(iter types.DocumentsIterator) int64 {
```

Comment:
@AlekSi do you understand why some lines are marked as not covered by tests? E.g. this function - the signature has changes, so it has changed in the callers too, but where are those callers?

Reply:
There are none. This function will be used in future PRs.